CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.2.1
```

Comment:
There's already a version 3.2.1 that has 

https://github.com/Workiva/dependency_validator/pull/85 Update references to the archived slack room #support-client-plat

Reply:
good catch, fixed!